Description: Fixed the undefined reference to blas_set_parameter 
Author: Dandan Zhang <zhangdandan@loongson.cn>
Applied-Upstream: <0.3.29>, <https://github.com/OpenMathLib/OpenBLAS/pull/4877> 
Last-Update: 2024-11-01

--- openblas-0.3.28+ds.orig/driver/others/blas_server_omp.c
+++ openblas-0.3.28+ds/driver/others/blas_server_omp.c
@@ -114,9 +114,11 @@ void goto_set_num_threads(int num_thread
 
   adjust_thread_buffers();
 #if defined(ARCH_MIPS64) || defined(ARCH_LOONGARCH64)
+#ifndef DYNAMIC_ARCH
   //set parameters for different number of threads.
   blas_set_parameter();
+#endif
 #endif
 
 }
 void openblas_set_num_threads(int num_threads) {
